.github/workflows/pr-go-e2e-all.yml

```diff
@@ -0,0 +1,16 @@
+name: CLI E2E tests
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+      - run: 'echo "No test run required"'
```

Comment:
Why do we need this in addition to the filtered one?

Reply:
For PR checks to pass. Annoying GitHub workaround lol